scipy/signal/filter_design.py

```diff
@@ -2298,6 +2298,8 @@ def iirfilter(N, Wn, rp=None, rs=None, btype='band', analog=False,
 
     fs : float, optional
         The sampling frequency of the digital system.
+        If it is specified then units of 'fs' and 'Wn' will be the same i.e. half-cycles/sample.
```

Comment:
Minor complaint from the CI tests: `scipy/signal/filter_design.py:2301:80: E501 line too long (96 > 79 characters)`

Reply:
Well, this is my first contribution.
Do I have to make another and then give another pull request ?